utils/misc.go

```diff
@@ -619,3 +619,42 @@ func GetDeploymentFilePath(projectPath string) string {
 	}
 }
 
+// agnostic util reader to fetch content from web or local path or potentially other places.
```

Comment:
Did we get some optimization for moving these to misc.go since they are still part of same package?  Is this just reducing # of files we have for simplicity?